bench-tps/src/main.rs

```diff
@@ -62,6 +60,7 @@ fn main() {
     if converge_only {
         return;
     }
+    let cluster_entrypoint = nodes[0].clone(); // Pick the first node, why not?
```

Comment:
Fix is here.   `ContactInfo::new_entry_point()` creates a stub ContactInfo that only contains a gossip address.  Here we now select a fully populated ContactInfo, which approximates what was done earlier.

I also renamed `ContactInfo::new_entry_point()` to `ContactInfo::new_gossip_entry_point()` to make to more clear of the limations